modules/ROOT/partials/configuration/invalid_elements.adoc

```diff
@@ -5,7 +5,6 @@ The `invalid_elements` option instructs the editor to remove specific elements w
 
 *Type:* `String`
 
-[discrete]
 === Example: Using `invalid_elements`
 
 [source, js]
```

Comment:
At the end of this file there are xref that need the full ref as they refer to things in other files:
Line 20: ```xref:valid_elements[`valid_elements`]``` and ```xref:extended_valid_elements[`extended_valid_elements`]```